src/mono/wasm/build/WasmApp.Common.targets

```diff
@@ -621,6 +621,8 @@
 
     <Error Condition="'$(RunAOTCompilation)' == 'true' and '$(PublishTrimmed)' != 'true'"
            Text="AOT is not supported without IL trimming (PublishTrimmed=true required)." />
+    <Error Condition="'$(RunAOTCompilation)' == 'true' and '$(Configuration)' == 'Debug'"
```

Comment:
We rely on `Configuration` variable value, comparing it with `Release` and `Debug` in multiple places.
https://github.com/dotnet/runtime/blob/27d1ab03931cba67583bea79900e89687be9eb02/src/mono/wasm/build/WasmApp.Common.targets#L554
https://github.com/dotnet/runtime/blob/27d1ab03931cba67583bea79900e89687be9eb02/src/mono/wasm/build/WasmApp.Common.targets#L546
https://github.com/dotnet/runtime/blob/27d1ab03931cba67583bea79900e89687be9eb02/src/mono/mono.proj#L495
Do you mean that there's a better property to check the configuration?

Reply:
I'm aware of other places (that they exist) and every now and then they fail for some user with custom config name (there possible not much of them, but some such users exist). So I would like to know if there is a better property to check and hence the question what is the true underlaying requirement for AOT compiler, so we could check that requirement instead. 

If the requirement is one of the places where we check for config==debug, there is nothing to do about it. If the requirement is something different with some concrete property value, it would be great to check that property.

Reply:
I see. The requirement: runtime without optimization (so debug-friendly) will take long time to build app with AOT and in the end it won't support managed debugging, so there was no point. Useful properties would be connected with managed debugging or optimization on/off.

Reply:
I am not able to find a good alternative